src/renderer/modals/SignTransaction/steps/StepSummary.js

```diff
@@ -0,0 +1,268 @@
+// @flow
```

Comment:
This component has been duplicated 1 to 1 from the send workflow as of today.
When we will rework the design of the sign transaction flow, we will be able to tailor it accordingly.